updater/lib/dependabot/service.rb

```diff
@@ -84,6 +86,42 @@ def error_summary
       "Dependabot encountered '#{errors.length}' error(s) during execution, please check the logs for more details."
     end
 
+    # Example output:
+    #
+    # +--------------------+
+    # |    Errors          |
+    # +--------------------+
+    # | job_repo_not_found |
+    # +--------------------+
+    def job_error_type_summary
+      job_error_types = errors.filter_map { |error_type, dependency| [error_type] if dependency.nil? }
+      return unless job_error_types.any?
+
+      Terminal::Table.new do |t|
+        t.title = "Errors"
+        t.rows = job_error_types
+      end
+    end
+
+    # Example output:
+    #
+    # +-------------------------------------+
+    # |    Dependencies failed to update    |
+    # +---------------------+---------------+
+    # | best_dependency_yay | unknown_error |
+    # +---------------------+---------------+
+    def dependency_error_summary
+      dependency_errors = errors.filter_map do |error_type, dependency|
+        [dependency.name, error_type] unless dependency.nil?
+      end
+      return unless dependency_errors.any?
```

Comment:
💅 Personal preference: _I_ find `unless` harder to think about. Another option would be to use `#none?` with `if` 👇

```suggestion
      return if dependency_errors.none?
```

If you see this similarly, and feel like making the change, `return unless job_error_types.any?` could use the same approach 😄